src/host/rcp_host.cpp

```diff
@@ -289,6 +289,10 @@ void RcpHost::Init(void)
 
     otBorderAgentSetMeshCoPServiceChangedCallback(mInstance, RcpHost::HandleMeshCoPServiceChanged, this);
     otBorderAgentEphemeralKeySetCallback(mInstance, RcpHost::HandleEpskcStateChanged, this);
+#if OTBR_ENABLE_BACKBONE_ROUTER
+    otBackboneRouterSetMulticastListenerCallback(mInstance, RcpHost::HandleBackboneRouterMulticastListenerChanged,
+                                                 this);
```

Comment:
So this PR modifies both RCP and NCP. The PR description should clarify this.

Reply:
I actually didn't update the implementation of RCP. That needs update the posix platform in openthread. Since the implementation for this ThreadHost API in RCP is short, I put it in this PR BTW.